modules/core/src/core/geometry.js

```diff
@@ -20,6 +22,21 @@ export function getDrawMode(drawMode) {
   return mode;
 }
 
+export function getBuffersFromGeometry(gl, geometry) {
```

Comment:
Since this knows about GL, which Geometry shouldn't, sould probably go into separate file?